src/gui/AstroCalcDialog.cpp

```diff
@@ -7207,7 +7207,7 @@ void AstroCalcDialog::calculateWutObjects()
 		ui->wutAngularSizeLimitMaxSpinBox->setToolTip(q_("Maximum angular size for visible celestial objects"));
 
 		// Direct calculate sunrise/sunset (civil twilight)
-		Vec4d rts = GETSTELMODULE(SolarSystem)->getSun()->getRTSTime(core, -7.);
+		Vec4d rts = GETSTELMODULE(SolarSystem)->getSun()->getRTSTime(core, -6.);
```

Comment:
I would call this differently, e.g. rtsM6 or rtsCivil, to make the code clearer.

Reply:
This variable re-used in other parts of this method

Reply:
Ah, OK.